src/plugins/range/CMakeLists.txt

```diff
@@ -1,5 +1,12 @@
 include (LibAddMacros)
 
+if (DEPENDENCY_PHASE)
+    set_source_files_properties(range.c PROPERTIES COMPILE_FLAGS -Wno-psabi)
+endif ()
+if(UNIX)
```

Comment:
Properly done you would need to check with [CHECK_FUNCTION_EXISTS](http://stackoverflow.com/questions/32816646/can-cmake-detect-if-i-need-to-link-to-libm-when-using-pow-in-c) if libm is needed for fpclassify.